lib/routes.js

```diff
@@ -53,6 +53,15 @@ app.use(nocache())
 app.use(express.json({ limit: '2mb' }))
 app.use(morgan(':method :url :status :response-time ms - :res[content-length]', { stream: logger.stream }))
 
+app.use('/robots.txt', (req, res) => {
+  res.type('text/plain')
+  res.send("User-agent: *\nDisallow: /")
+})
+
+app.get('/', (req, res) => {
+  res.status(404).json({ error: 'No such route' })
```

Comment:
No need to return the json here. The more generic the better, so just 404 should be enough.